haskell/nix.bzl

```diff
@@ -0,0 +1,172 @@
+"""Loading of haskell packages from nixpkgs"""
+
+load(
+    "@io_tweag_rules_nixpkgs//nixpkgs:nixpkgs.bzl",
+    "nixpkgs_package",
+)
+
+def haskell_nixpkgs_package(
+        name,
+        attribute_path,
+        nix_file_deps = [],
+        repositories = {},
+        build_file_content = None,
+        build_file = None,
+        *args,
```

Comment:
I don't think we need to support non-keyword arguments here. It would be bad style if the user did that.